### PR TITLE
Search name feature

### LINKED ISSUE #11: Search scientific name of an animal/plant local name
Given a local name (or maybe the location) of an animal/plant, get the scientific name and its corresponding information.

### PR DESCRIPTION
**Pull request by rydwhelchel**
Added the feature for requesting a search for a name which is not scientific. Uses the GBIF database as a reference. Just add a `-n` tag to the end of the input line. 

Closes #11

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -44,6 +44,23 @@ def getGBIFSearch(query):
...
+    }
+    response = requests.get('http://api.gbif.org/v1/species/search', params=args)
+    if response.status_code == 200:
+      data = response.json()
+      for result in data['results']:
+        try:
+            name = result['species']
+            return name
+        except KeyError:
+            continue
```
Comment (rizqirizqi) [inline main.py:61]: using safe navigation is preferred:

```py
for result in data['results']:
  species = result.get('species')
  if species:
    return species
```

**Comment by rydwhelchel** [comment]
I fixed safe navigation. I would consider this feature a work in progress for now. It will still miss a lot of searches. I have an idea to add more arguments in the future, perhaps `-a` for Animal Kingdom and `-p` for Plant Kingdom. There also is probably a better way to handle these arguments than appending tags to the end of input lines.

**Comment by rizqirizqi** [comment]
I agree, using tags like that seems to be not really intuitive. It also requires the user of this tool to append the tags to their input first before using this tool, which sometimes they may have a thousand names and they also don't really know which name is a scientific name, a plant, or an animal. And this tool is supposed to be helping them to identify those names.

So, do you want to modify this PR first or want it to be merged for now? I'm okay with either way.

Anyway, thanks for the PR! Feel free to add more features. 💚

**Review by rizqirizqi — APPROVED** [review]

**Comment by rydwhelchel** [comment]
I think go ahead and merge, but I probably will come back to this issue in the future.

**Pull request merged**